src/mpris.h

```diff
@@ -44,9 +44,12 @@ class AMBER_MPRIS_EXPORT Mpris
     Q_ENUM(PlaybackStatus);
 
     enum LoopStatus {
-        None,
-        Track,
-        Playlist
+        LoopNone,
+        LoopTrack,
+        LoopPlaylist,
+        None = LoopNone, // this and the following deprecated
+        Track = LoopTrack,
+        Playlist = LoopPlaylist
```

Comment:
These compatibility values probably keep `MprisPlayerPrivate::setLoopStatus()` afloat, but `MprisPlayerPrivate::loopStatus` probably starts to spew out the wrong strings.

Reply:
Separated the d-bus strings and the corresponding enums there.